CHANGES.txt

```diff
@@ -1,3 +1,65 @@
+2016-09-23 version 3.1.0 (C++/Java/Python/PHP/Objective-C)
+  General
+  * Proto3 support in PHP.
+  * Various bug fixes.
+
+  C++
```

Comment:
Also there are performance optimizations to avoid static allocation. Gerben can probably add something for that.